components/feature/pwa/src/main/java/mozilla/components/feature/pwa/WebAppLauncherActivity.kt

```diff
@@ -76,16 +77,14 @@ class WebAppLauncherActivity : AppCompatActivity(), CoroutineScope by MainScope(
 
     @VisibleForTesting(otherwise = PRIVATE)
     internal fun launchWebAppShell(startUrl: Uri) {
-        val intent = Intent(AbstractWebAppShellActivity.INTENT_ACTION).apply {
-            data = startUrl
-            flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_MULTIPLE_TASK
+        val intent = Intent(ACTION_VIEW_PWA, startUrl).apply {
             `package` = packageName
         }
 
         try {
             startActivity(intent)
         } catch (e: ActivityNotFoundException) {
-            logger.error("Packages does not handle AbstractWebAppShellActivity intent. Can't launch web app.", e)
+            logger.error("Packages does not handle ACTION_VIEW_PWA intent. Can't launch as web app.", e)
```

Comment:
I'm not sure if the intention was `$ACTION_VIEW_PWA`?

Reply:
This is actually from #3816.
Browsers no longer need to subclass `AbstractWebAppShellActivity` for PWA support. The `WebAppIntentProcessor` checks for ACTION_VIEW_PWA instead.